lib/everest.cpp

```diff
@@ -188,8 +188,8 @@ Everest::Everest(std::string module_id_, const Config& config_, bool validate_da
     }
 
     // register handler for global ready signal
-    const Handler handle_ready_wrapper = [this](json data) { this->handle_ready(data); };
-    const std::shared_ptr<TypedHandler> everest_ready =
```

Comment:
why have all the `const`s disappeared?  `auto` can also be used for the `shared_ptr` variables

Reply:
Looks like that's from the recent merge of main where this turned from `Handler` to `auto`. I'll change that